sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/PerfCounterItemCounts.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    /// <summary>
+    /// Thread-safe counters for performance metrics.
+    /// </summary>
+    internal sealed class PerfCounterItemCounts
+    {
+        private long _totalRequestCount = 0;
+        private long _totalExceptionCount = 0;
+
+        /// <summary>
+        /// Atomically increments the total request count.
+        /// </summary>
+        /// <returns>The new value after incrementing.</returns>
+        public long IncrementRequestCount()
+        {
+            return Interlocked.Increment(ref _totalRequestCount);
```

Comment:
Request rate should be derived from `http.server.request.duration` metrics by taking the histogram count and dividing by 60 (the aggregation window) to get requests per second.